libs/components/indicators/src/lib/modules/icon/icon-stack.component.ts

```diff
@@ -1,11 +1,12 @@
-import { Component, Input } from '@angular/core';
+import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
 
 import { SkyIconStackItem } from './icon-stack-item';
 
 @Component({
   selector: 'sky-icon-stack',
   templateUrl: './icon-stack.component.html',
   styleUrls: ['./icon-stack.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
```

Comment:
Enabling `OnPush` change detection at the leaves of the change detection tree (the root was the Alert component).

Reply:
I don't love doing this - a change in detection strategy could potentially cause issues. I understand that we should be using `OnPush` for new things and may need to look at old at times - but I don't love doing it just for this personally.